gallery/base.html

```diff
@@ -0,0 +1,222 @@
+                  </ul>
+                </li>
+                <li class="has-children"><a href="#">Subject Notes</a>
+                  <ul class="dropdown">
+                    <li><a href="https://drive.google.com/open?id=1UNN6vWwD3OghWKXXYk5jsrPC-esnzAtt">First Year</a></li>
+                    <li><a href="https://drive.google.com/open?id=1lP76R5LvfFHVYH5yQlC2aZFOaIqu2yV9">Second Year</a></li>
+                    <li><a href="https://drive.google.com/open?id=1UqkfxmiAw5hJgbcZFW_mWRGHYW8uVp-j">Third Year</a></li>
+                  </ul>
+                </li>
+            </li>
+            <li class="has-children"><a href="#">Past Pappers</a>
+              <ul class="dropdown">
+                <li><a href="https://drive.google.com/open?id=1xhNMyx-A7Twg9XdsF6EZsmX-H8KaJoGq">First Year</a></li>
+                <li><a href="https://drive.google.com/open?id=1HOfMim4Fit6gRWh-8eEOXzNrHy097kTw">Second Year</a></li>
+                <li><a href="https://drive.google.com/open?id=1ZPqZOYGo-uRwsqgiX_VldQqgwFHGu7iU">Third Year</a></li>
+              </ul>
+            </li>
+            </li>
+          </ul>
+          </li>
+          </ul>
+        </nav>
+      </div>
+
+      <div class="col-6 col-xl-2 text-right" data-aos="fade-down">
+        <div class="d-none d-xl-inline-block">
+          <ul class="site-menu js-clone-nav ml-auto list-unstyled d-flex text-right mb-0" data-class="social">
+            <li>
+              <a href="https://www.facebook.com/spec.ece/" class="pl-0 pr-3"><span class="icon-facebook"></span></a>
+            </li>
+            <li>
+              <a href="https://twitter.com/SPEC__NITH" class="pl-3 pr-3"><span class="icon-twitter"></span></a>
+            </li>
+            <li>
+              <a href="https://www.instagram.com/s.p.e.c_nith/" class="pl-3 pr-3"><span class="icon-instagram"></span></a>
+            </li>
+            <li>
+              <a href="https://in.linkedin.com/in/s-p-e-c-nith-40214b197" class="pl-3 pr-3"><span class="icon-linkedin"></span></a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="d-inline-block d-xl-none ml-md-0 mr-auto py-3" style="position: relative; top: 3px;"><a href="#" class="site-menu-toggle js-menu-toggle text-black"><span class="icon-menu h3"></span></a></div>
+
+      </div>
+
+    </div>
+  </div>
+
+</header>
+
+
+
+    <div class="container-fluid" data-aos="fade" data-aos-delay="500">
+      <div class="swiper-container images-carousel">
+        <div class="swiper-wrapper">
+          <div class="swiper-slide">
+            <div class="image-wrap">
+              <div class="image-info">
+                <h2 class="mb-3" id="selector_text">SPEC Fest</h2>
+                <a href="fest.html" class="btn btn-outline-white py-2 px-4">More Photos</a>
+              </div>
+              <img src="bg-img/img_1.jpg" alt="Image">
```

Comment:
@sjainit @baamaboi  make use of backslash for paths. Do the same for pages rest of the pages you worked on,

Reply:
Backslash are only used in Windows and nowhere else, most server use Linux. Web browsers as well as network protocols use Unix conventions.